virt/virtual_machines/virt-installing-qemu-guest-agent.adoc

```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="virt-installing-qemu-guest-agent"]
-= Installing the QEMU guest agent on virtual machines
+= Installing the QEMU guest agent and VirtIO drivers
 include::_attributes/common-attributes.adoc[]
 :context: virt-installing-qemu-guest-agent
 
```

Comment:
Just a suggestion. I think the TOC would be easier to use if you were to add 2 level 2 headers:

== Installing the QEMU guest agent
Followed by "include" files with 'leveloffset=+2'

== Install VirtIO drivers
Again, adjusting the level offset of the "include" files.